SplunkRumWorkspace/SplunkRum/SplunkRumTests/TestUtils.swift

```diff
@@ -100,7 +100,10 @@ func initializeTestEnvironment() throws {
     options.environment = "env"
     options.ignoreURLs = try! NSRegularExpression(pattern: ".*ignore_this.*")
 
-    SplunkRum.initialize(beaconUrl: "http://127.0.0.1:8989/v1/traces", rumAuth: "FAKE", options: options)
+    let rumInitialize = SplunkRum.initialize(beaconUrl: "http://127.0.0.1:8989/v1/traces", rumAuth: "FAKE", options: options)
+    let isRUMInitialized = SplunkRum.isInitialized()
+    let initializeAgain = SplunkRum.initialize(beaconUrl: "http://127.0.0.1:8989/v1/traces", rumAuth: "FAKE", options: options)
+    let isStillInitialized = SplunkRum.isInitialized()
```

Comment:
Could perhaps add the asserts here right away

Reply:
Added